client/asset/dcrspvclient/client.go

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) 2022-2023 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+// TODO: Files copied from decred/vspd. Delete this package and import from
+// decred/vspd when possible. That will probably be when the replaces there are
+// gone and this repo is using dcrwallet/v3.
+
+package dcrspvclient
```

Comment:
I think you can get in dcrwallet/v3 now.  Hopefully code duplication is mostly gone now.

Also, I'm scratching my head over dcr**spv**client.  Should that have been dcr**vsp**client?

Reply:
> Should that have been dcrvspclient?

yes